test/mock/honeybadger.go

```diff
@@ -0,0 +1,18 @@
+package mock
+
+import (
+	"github.com/DE-labtory/cleisthenes"
+	"github.com/DE-labtory/cleisthenes/pb"
+)
+
+type HoneyBadger struct {
```

Comment:
What is this package for?
`queue_internal_test.go` use internal `MockHoneyBadger`

Reply:
not used. I'll delete it!